Sources/Player/Common/Event/NotificationsHandler.swift

```diff
@@ -2,10 +2,12 @@ import Foundation
 
 final class NotificationsHandler {
 	private let listener: PlayerListener
```

Comment:
[question] WDYT about renaming this to `playerListener`, since now we have more than one "listener" in the class.

Reply:
Yeah, I had though about that! 
I will do it.